debian.oracle-5.15/config/delphix_annotations

```diff
@@ -0,0 +1,232 @@
+CONFIG_QLCNIC                                   policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_RDS                                      policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_REALTEK_PHY                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCIF                                     policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_ADVANSYS                            policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_AIC7XXX                             policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_AIC94XX                             policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_ARCMSR                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_BFA_FC                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_ESAS2R                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_GDTH                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_HPSA                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_IPS                                 policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_LPFC                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_MVSAS                               policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_MYRB                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_MYRS                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_PM8001                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_PMCRAID                             policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_QLA_FC                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_SMARTPQI                            policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_SNIC                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_SYM53C8XX_2                         policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SCSI_UFSHCD                              policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SENSORS_LM93                             policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SFC                                      policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SOUND                                    policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SSB                                      policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_STMMAC_ETH                               policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SXGBE_ETH                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_SYNCLINK                                 policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_USB                                      policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_VBOXGUEST                                policy<{'amd64': 'n', 'arm64': 'n'}>
+CONFIG_VOP                                      policy<{'amd64': 'n', 'arm64': 'n'}>
+
+include "../../debian.oracle-5.15/config/delphix_annotations"
```

Comment:
shd this be here? self referential file?

Reply:
Fixed.